master/buildbot/changes/gitpoller.py

```diff
@@ -24,10 +25,14 @@
 
 from buildbot import config
 from buildbot.changes import base
-from buildbot.util import ascii2unicode
+from buildbot.util import ascii2unicode, asyncSleep
```

Comment:
I just looked again at the link you gave me. It says that when using the `from foo import bar` type declarations, it is okay to make more than one import per line.

> It's okay to say this though:
```
from subprocess import Popen, PIPE
```

Should I still change it?